events/events-team/comms/README.md

```diff
@@ -1,55 +1,191 @@
+- Basic familiarity with the [Hugo website framework](https://gohugo.io/)
+- Basic familiarity with the [Markdown](https://www.markdownguide.org/) markup language
+
+### Tasks
+During the course of their KCS cycle as a website manager, KCS Website Managers are expected to complete tasks including but not limited to:
+- Joining the appropriate Slack channel(s) for KCS staff collaboration
+- Creating the page for the specific event within k8s.dev/events
+- Adding and maintaining information and pages throughout the event cycle, including:
+  - Registration info
+  - CFP & schedule info
+  - Social venue and registration info
+  - and more
+
+## Emeritus KCS Comms Lead Overview
+After completing a cycle as a KCS Comms Lead, the former lead takes on emeritus status. As an Emeritus KCS Lead, the former lead is expected to mentor the following KCS Lead(s). The Emeritus Lead can provide mentorship for as long as they like, but it's especially valuable for the previous lead to mentor their direct successor.
+
+### Requirements
+- Has completed at least 1 cycle as a KCS Comms Lead
+- [Kubernetes Org Membership]
+
+### Tasks
+- Provide mentorship to the KCS Comms Lead
+
+## Comms Rules and Guidelines
+
+### Social Media Rules
+- Send KCS comms to k8scontributors accounts on whatever platforms they exist on. At time of writing, that’s “X” and Mastodon. Social media should be sent only to contributor-facing accounts.
+  - Kubernetesio on X and the Kubernetes Linkedin accounts should not be used because their audiences include end users and the general public..
+- Make sure comms are formatted in Arial, 11pt, with single line spacing before copying into the target platform. This will ensure that the text looks consistent wherever it is posted.
+- If you are scheduling comms to go out at a specific time, make sure that time is aligned with the timezone the event will be held in.
+- Link to k8s.dev/events and subpages, or email messages, wherever possible. Avoid sharing links to contributor-specific links such as Registration, CFP, and GitHub Issues.
+  - While our Social Media presence is largely targeted at contributors, all social media posts are public. As such, we should avoid sharing links which could cause unwanted spam/interactions.
+- Review the Contributor Comms team’s Social Media Guidelines.
+- All social media should have at least 1 review, preferably by a shadow. A review by a social media approver is a given.
+- Generally, social media should be ready for review at least 1 day before it needs to be sent.
+- Generally, an email should be ready for review around 3 or more days before it needs to be sent.
+- If any shadow does not have Buffer access, let the KCS Comms lead know.
+- Always consider your intended and actual audience! When a message is only applicable to org members (which is pretty much - everything for the contributor summit), state that clearly!
+
```

Comment:
```suggestion
```
(nit - an extra line) Not sure if it'd impact the final markdown version so pointing it out just in case :)